rts/Rendering/Units/UnitDrawer.cpp

```diff
@@ -508,13 +508,15 @@ void CUnitDrawerLegacy::DrawUnitIconsScreen() const
 
 
 	const float3 camDir = camera->GetDir();
-	// We are going to clamp camDir.y at -0.5, then re-normalize it. 
-	const float unitIconCameraMinY = configHandler->GetFloat("UnitIconCameraMinY");
+
+	// Grab the minimum camera tilt, below which we use global map-camera distance, above which we use  individual unit-camera distances
+	const float unitIconCameraMinAngleY = std::sin(configHandler->GetFloat("UnitIconCameraMinAngleY") * math::DEG_TO_RAD);
```

Comment:
Perhaps this value could be cached somehow (with a listener for the "config changed" event)? Since it doesn't change but is used 2 times per draw frame.

Reply:
I hate how you are right.